tests/common/experimental/test_statistic_collector.py

```diff
@@ -333,10 +322,6 @@ def test_empty_tensors_register(self, inplace, any_not_empty):
             {input_name: self.get_nncf_tensor(np.array([]))}, [(hash(reducer), [input_name])]
         )
 
-        stats = collector.get_statistics()
-        assert len(stats) == 1
-        assert stats["A"] is None
-
```

Comment:
This check should work ok, why it was removed here?

Reply:
Just keep this three line and add `collector.enable()` afterwards

Reply:
your suggestion does not work

Reply:
updated as discussed